packages/eds-core-react/build/index.css

```diff
@@ -0,0 +1,122 @@
+/* Styles for Next EDS 2.0 Components (vanilla CSS) */
+
+@layer eds-components {
+  /* Container - use neutral appearance by default */
+  .eds-input {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    box-sizing: border-box;
+    width: 100%;
+    background: var(--eds-color-bg-canvas);
```

Comment:
We actually have a unique background for input in our "concept" tokens: https://github.com/equinor/design-system/blob/main/packages/eds-tokens/build/css/variables.css#L12